fetch_models.js

```diff
@@ -0,0 +1,163 @@
+      path: "./demos/stable-diffusion-1.5/models"
+    },
+    {
+      url: "microsoft/stable-diffusion-v1.5-webnn/resolve/main/Stable-Diffusion-v1.5-vae-decoder-float16-fp32-instancenorm.onnx",
+      path: "./demos/stable-diffusion-1.5/models"
+    },
+    {
+      url: "microsoft/stable-diffusion-v1.5-webnn/resolve/main/safety-checker.onnx",
+      path: "./demos/stable-diffusion-1.5/models"
+    },
+    {
+      url: "microsoft/segment-anything-model-webnn/resolve/main/sam_vit_b_01ec64.encoder-fp16.onnx",
+      path: "./demos/segment-anything/models"
+    },
+    {
+      url: "microsoft/segment-anything-model-webnn/resolve/main/sam_vit_b_01ec64.decoder-fp16.onnx",
+      path: "./demos/segment-anything/models"
+    },
+    {
+      url: "microsoft/segment-anything-model-webnn/resolve/main/sam_vit_b-encoder-int8.onnx",
+      path: "./demos/segment-anything/models"
+    },
+    {
+      url: "microsoft/segment-anything-model-webnn/resolve/main/sam_vit_b-decoder-int8.onnx",
+      path: "./demos/segment-anything/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_decoder_static_kvcache_128_lm_fp16_layernorm.onnx",
+      path: "./demos/whisper-base/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_decoder_static_kvcache_128_lm_fp16_layernorm_4dmask.onnx",
+      path: "./demos/whisper-base/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_decoder_static_kvcache_128_lm_fp16_layernorm_gelu_4dmask.onnx",
+      path: "./demos/whisper-base/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_decoder_static_non_kvcache_lm_fp16_layernorm.onnx",
+      path: "./demos/whisper-base/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_decoder_static_non_kvcache_lm_fp16_layernorm_4dmask.onnx",
+      path: "./demos/whisper-base/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_decoder_static_non_kvcache_lm_fp16_layernorm_gelu_4dmask.onnx",
+      path: "./demos/whisper-base/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_encoder_lm_fp16_layernorm.onnx",
+      path: "./demos/whisper-base/models"
+    },
+    {
+      url: "microsoft/whisper-base-webnn/resolve/main/whisper_base_encoder_lm_fp16_layernorm_gelu.onnx",
+      path: "./demos/whisper-base/models"
+    }
+  ];
+
+  const downloadFile = async (url, outputPath, retries = 2) => {
+    // // Proxy configuration
+    // const proxyHost = '';
+    // const proxyPort = 33630;
+    // const proxyUrl = `http://${proxyHost}:${proxyPort}`;
+    // const proxyAgent = new HttpsProxyAgent(proxyUrl);
+
+    try {
+      // const res = await fetch(url, {agent: proxyAgent});
+      const res = await fetch(url);
```

Comment:
```suggestion
      const resource = await fetch(url);
```
OR
```suggestion
      const response = await fetch(url);
```
whol wrds are bttr.

Reply:
Indeed, update `res` to `response` now.